src/transformers/__init__.py

```diff
@@ -1879,6 +1880,9 @@
     _import_structure["models.rag"].extend(
         ["RagModel", "RagPreTrainedModel", "RagSequenceForGeneration", "RagTokenForGeneration"]
     )
+    _import_structure["models.atlas"].extend(
+        ["AtlasModel", "AtlasPreTrainedModel", "AtlasSequenceForGeneration", "AtlasTokenForGeneration"]
```

Comment:
These model names are a bit strange. Might be inspired from rag but we dropped these kind of notation. Also from reading the paper it seems that you need `AtlasForConditionalGeneration` (as it follows the T5 formatting of prompts) ans `AtalsForMaskedLM` for training/fine tuning the model. This comment is valid for the rest of the `__init__` files and of course the modelling